src/main/java/tests/YouTubeTest.java

```diff
@@ -1,54 +1,37 @@
 package tests;
 
+import configreader.ConfigManager;
 import driverfactory.DriverFactory;
 import enums.DriverNames;
 import org.junit.AfterClass;
 import org.junit.BeforeClass;
 import org.junit.Test;
 import org.openqa.selenium.WebDriver;
 import pages.YouTubePage;
+
 import java.io.IOException;
+import java.util.concurrent.TimeUnit;
 
 public class YouTubeTest {
-    private static WebDriver webDriver;
-
+    private static WebDriver driver;
+    private static YouTubePage youTubePage;
 
     @BeforeClass
     public static void openBrowser() throws IOException {
-        System.out.println("Starting test.");
-    }
-
-    @Test
-    public void startChromeTest() throws IOException {
-        webDriver = DriverFactory.getDriver(DriverNames.CHROME);
-        YouTubePage youTubePage = new YouTubePage(webDriver);
-        youTubePage.startTest();
-    }
-
-    @Test
-    public void startOperaTest() throws IOException {
-        webDriver = DriverFactory.getDriver(DriverNames.OPERA);
-        YouTubePage youTubePage = new YouTubePage(webDriver);
-        youTubePage.startTest();
+        driver = DriverFactory.getDriver(DriverNames.CHROME);
+        driver.manage().timeouts().implicitlyWait(2, TimeUnit.SECONDS);
```

Comment:
the implicit wait should be set only once, usually it is done in Driver Factory, not in tests